homeassistant/components/modbus/base_platform.py

```diff
@@ -76,6 +76,12 @@ class BasePlatform(Entity):
 
     def __init__(self, hub: ModbusHub, entry: dict[str, Any]) -> None:
         """Initialize the Modbus binary sensor."""
+
+        if CONF_LAZY_ERROR in entry:
+            _LOGGER.warning(
+                "`lazy_error_count`: is deprecated and will be removed in version 2024.5"
+            )
```

Comment:
✨ issues ✨

Also, code still says 2024.5, PR says 2024.6, while this will probably go into 2024.1, so it should be 2024.7

Reply:
Ups, good catch...I will update the PR in a moment

Reply:
I also meant, please consider raising an issue instead of only logging a warning

Reply:
I did consider it, as written below in a comment..

But it would be a lot bigger change, because I need to transport hass through a number of methods in order to make it available in base_platform.py, so I opted out of the this option, to keep the PR simple.

Reply:
@joostlek it x-mas very soon, so your wish came through.

I have added create_issue.